### PR TITLE
CEDS-3607 - Content updates

### PR DESCRIPTION
**Pull request by oddbunchmedia**

### CODE REVIEW

File: app/views/declaration/carrier_eori_number.scala.html [hunk 1]
```diff
@@ -34,6 +34,8 @@
     errorSummary: errorSummary,
     sectionHeader: sectionHeader,
     heading: heading,
+    insetText: exportsInsetText,
+    paragraphBody: paragraphBody,
     body: paragraphBody,
```
Comment (udinobi) [inline app/views/declaration/carrier_eori_number.scala.html:39]: Why 2 paragraphBody instances? You only need to specify it one time.

File: app/views/components/gds/exportsInputText.scala.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
     hintKeyLinks: Seq[Html] = Seq.empty,
     hintHtml: Option[Html] = None,
     bodyHtml: Option[Html] = None,
-    labelClasses: String = "govuk-label govuk-label--m",
+    labelClasses: String = "",
```
[the remark this answers is not in the document]
Comment (oddbunchmedia) [reply app/views/components/gds/exportsInputText.scala.html:33]: I thought this initially but it is set elswehere as I ended up with class="govuk-label govuk-label".

File: app/views/declaration/carrier_eori_number.scala.html (hunk 1 above)
Comment (oddbunchmedia) [reply app/views/declaration/carrier_eori_number.scala.html:39]: Arghh, didn't notice the body: paragraphBody... :(

File: app/views/components/gds/exportsInputText.scala.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (udinobi) [reply app/views/components/gds/exportsInputText.scala.html:33]: :thinking:

File: app/views/declaration/carrier_eori_number.scala.html (hunk 1 above)
Comment (oddbunchmedia) [reply app/views/declaration/carrier_eori_number.scala.html:38]: Fixed

File: app/views/components/gds/exportsInputText.scala.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (oddbunchmedia) [reply app/views/components/gds/exportsInputText.scala.html:33]: Screenshot from 2021-11-30 16-16-10

**Review by udinobi — APPROVED** [review]

**Pull request merged**